### PR TITLE
Remove image builder

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
- Removed image builder and all its dependencies

## QA
- Go to /core/build and see its deleted
- Check site login still works
- Check the /advantage section still looks ok

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/3088

### CODE REVIEW

File: webapp/views.py [hunk 1]
```diff
@@ -1,7 +1,4 @@
 # Standard library
-import hashlib
-import hmac
-import json
```
Comment (pmahnke) [inline webapp/views.py:4]: I think removing json has broken usn's and cve's?

File: requirements.txt [hunk 2]
```diff
@@ -6,10 +6,8 @@ canonicalwebteam.search==1.0.0
 canonicalwebteam.templatefinder==1.0.0
 canonicalwebteam.image-template==1.3.1
 canonicalwebteam.discourse==4.0.4
-canonicalwebteam.launchpad==0.8.2
```
Comment (nottrobin) [inline requirements.txt:9]: Pretty sure other things are gonna need this

File: webapp/views.py (hunk 1 above)
Comment (nottrobin) [reply webapp/views.py:4]: did it? Where did you see that?

**Review by nottrobin — APPROVED** [review]
This is really thorough! Looks good to me. Check @pmahnke agrees before merging, but I don't personally see anything wrong with removing `json` which doesn't seem to be used in `webapp/views.py`.

**Comment by anthonydillon** [comment]
Yeah looks like CVEs and USNs are blocked from loading on demos.

**Pull request merged**